corehq/apps/hqcase/tasks.py

```diff
@@ -22,11 +23,13 @@ def explode_cases(user_id, domain, factor, task=None):
     old_to_new = dict()
     child_cases = list()
 
+    case_ids = get_case_ids_in_domain_by_owner(
```

Comment:
That makes sense to me. Especially if it's more efficient on larger domains to grab ids first. I do like the idea of using function names as a way of convention rather than always specifying lots of args. The only thing is I'd probably be more verbose about the wrapped name just because to the new person it could be ambiguous, but I could either way. Perhaps they could all map to a single function that takes the args so we don't repeat the view logic. Anyways, for this PR I'm happy with what you have.